src/NServiceBus.Testing.Fakes/TestableOutgoingContext.cs

```diff
@@ -17,7 +17,9 @@ public partial class TestableOutgoingContext : TestablePipelineContext, IOutgoin
         /// </summary>
         public FakeBuilder Builder { get; set; } = new FakeBuilder();
 
-        IBuilder IBehaviorContext.Builder { get; }
+        IBuilder IBehaviorContext.Builder {
+            get { return this.Builder; }
```

Comment:
this is already fixed on latest develop. Best to revert and rebase (it should call GetBuilder() to enable users to provide their custom IBuilder).